code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -296,16 +295,16 @@
 
 		if(/obj/item/clothing/head/pirate, /obj/item/clothing/head/collectable/pirate)
 			name = "[pick("Ol'","Scurvy","Black","Rum","Gammy","Bloody","Gangrene","Death","Long-John")] [pick("kibble","leg","beard","tooth","poop-deck","Threepwood","Le Chuck","corsair","Silver","Crusoe")]"
-			desc = "Yaarghh!! Thar' be a scurvy dog!"
-			emote_see = list("hunts for treasure","stares coldly...","gnashes his tiny corgi teeth")
-			emote_hear = list("growls ferociously", "snarls")
-			speak = list("Arrrrgh!!","Grrrrrr!")
+			desc = "Yaarghh! Thar' be a scurvy dog!"
+			emote_see = list("hunts for treasure.","stares coldly...","gnashes his tiny corgi teeth.")
+			emote_hear = list("growls ferociously.", "snarls.")
+			speak = list("Arrrrgh!","Grrrrrr!")
 			valid = 1
 
 		if(/obj/item/clothing/head/ushanka)
-			name = "[pick("Comrade","Commissar","Glorious Leader")] [real_name]"
+			name = "[pick("товарищ","Vladmir","Chairman")] [real_name]"
```

Comment:
Bad idea to have cyrillic

Reply:
Sure, I can remove it. Is there any reason why? I know that the Russian mobs chant in cyrillic so I thought it would be fine.

Reply:
Goonchat doesn't play nice with non ascii.